librz/core/cmd_type.c

```diff
@@ -208,7 +198,7 @@ static void __core_cmd_tcc(RzCore *core, const char *input) {
 		pj_free(pj);
 		rz_list_free(list);
 	} break;
-	case 'l': {
+	case RZ_OUTPUT_MODE_LONG: {
```

Comment:
I don't see any command that you converted that accept the mode `RZ_OUTPUT_MODE_LONG`.

Reply:
Good point, added. `tccl` shoudl work now.